apps/openchallenges/challenge-service/src/main/java/org/sagebionetworks/openchallenges/challenge/service/model/repository/CustomChallengeRepositoryImpl.java

```diff
@@ -256,10 +256,51 @@ private SearchPredicate getChallengeIncentivesPredicate(
    */
   private SearchPredicate getCategoriesPredicate(
       SearchPredicateFactory pf, ChallengeSearchQueryDto query) {
+
+    LocalDate now = LocalDate.now();
+    LocalDate lastQuarter = now.minusMonths(3);
```

Comment:
> A quarter is a three-month period on a company's financial calendar that acts as a basis for periodic financial reports and the paying of dividends.

The variable name may introduce confusion as quarter are not any three-month periods but correspond to specific months (e.g. Q1 is often Jan-March).

Maybe `recentPeriod` and `soonPeriod`

Reply:
I agree there will be confusion. 
I changed to use "threeMonthsAgo" and "threeMonthsLater" to be more specific and clear.